android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireActivity.kt

```diff
@@ -147,20 +146,27 @@ open class QuestionnaireActivity : BaseMultiLanguageActivity(), View.OnClickList
             }
           }
         questionnaireResponse =
-          questionnaireViewModel.getQuestionnaireResponseFromDbOrPopulation(
-              questionnaire = questionnaire,
-              subjectId = baseResourceId.extractLogicalIdUuid(),
-              subjectType = baseResourceType,
-              questionnaireConfig = questionnaireConfig
-            )
-            .apply { generateMissingItems(questionnaire) }
+          if ((baseResourceId.isNullOrEmpty() || baseResourceType == null) &&
+              questionnaireConfig.type.isDefault()
+          ) {
```

Comment:
@dubdabasoduba I think this is why you are unable to load expressions, try giving questionnareType.Edit in your case

Reply:
I see